be/src/vec/exec/format/json/new_json_reader.cpp

```diff
@@ -1730,4 +1726,52 @@ Status NewJsonReader::_simdjson_write_columns_by_jsonpath(
     return Status::OK();
 }
 
+Status NewJsonReader::_get_column_default_value(
```

Comment:
we should `_fill_missing_column`  also when `fill missing slot`

```
 // fill missing slot
    int ctx_idx = 0;
    int nullcount = 0;
    for (auto slot_desc : slot_descs) {
        if (!slot_desc->is_materialized()) {
            continue;
        }
        int dest_index = ctx_idx++;
        auto* column_ptr = block.get_by_position(dest_index).column->assume_mutable().get();
        if (column_ptr->size() < cur_row_count + 1) {
           // ..._fill_missing_column.. here
            ++nullcount;
        }
        DCHECK(column_ptr->size() == cur_row_count + 1);
    }
    // There is at least one valid value here
    DCHECK(nullcount < block.columns());
    *valid = true;
```

Reply:
I think I have filled all slot before, do I need to fill missing slot again?

Reply:
so this logic is useless now? or maybe we could delete is?

Reply:
Yea, it's useless logically speaking

Reply:
> so this logic is useless now? or maybe we could delete it?

Yea, it's useless logically speaking